spec/models/early_career_payments/slug_sequence_spec.rb

```diff
@@ -74,17 +74,17 @@
       end
 
       [
-        {itt_subject: "Mathematics", itt_academic_year: "2019 - 2020"},
-        {itt_subject: "Mathematics", itt_academic_year: "2020 - 2021"},
-        {itt_subject: "Physics", itt_academic_year: "2020 - 2021"},
-        {itt_subject: "Chemistry", itt_academic_year: "2020 - 2021"},
-        {itt_subject: "Foreign languages", itt_academic_year: "2020 - 2021"}
+        {itt_subject: "mathematics", itt_academic_year: "2019/2020"},
```

Comment:
Maybe `AcademicYear` again?

Reply:
@nomasprime As per previous comment, Yes I think it would be good to be consistent